modules/gateway/persist.go

```diff
@@ -19,30 +19,81 @@ const (
 // gateway persist file.
 var persistMetadata = persist.Metadata{
 	Header:  "Sia Node List",
+	Version: "0.4.0",
+}
+
+// previousPersistMetadata contains the header and version strings that
+// identify the previous gateway persist file. It allows us to load files in a
+// backward-compatible way.
+var previousPersistMetadata = persist.Metadata{
```

Comment:
Again, don't think a version change is necessary here, but if it was, the correct name for this struct would be `v033PersistMetadata`